app/helpers/application_helper/button/smart_state_scan.rb

```diff
@@ -0,0 +1,19 @@
+class ApplicationHelper::Button::SmartStateScan < ApplicationHelper::Button::Basic
+  needs :@record
+
+  SMART_ROLES = %w(SmartProxy SmartState).freeze
```

Comment:
We are using this button for the VM instances too so this should be in a general location. Is there a central SSA class?